proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -101,6 +101,14 @@ message MsgCreateBatchRequest {
   google.protobuf.Timestamp end_date = 6
       [(gogoproto.stdtime) = true, (gogoproto.moretags) = "yaml:\"end_date\""];
 
+  // project_location is the location of the project backing the credits in this
+  // batch. It is a string of the form
+  // <country-code>[-<sub-national-code>[ <postal-code>]], with the first two
+  // fields conforming to ISO 3166-2, and postal-code being up to 64
+  // alphanumeric characters. country-code is required, while sub-national-code
+  // and postal-code can be added for increasing precision.
```

Comment:
same comments as above. Maybe we should document it here, and in other places note that the documentation of the field is in `MsgCreateBatchRequest.project_location` to avoid copy pasting and desyncing?